src/css/states/states.less

```diff
@@ -97,8 +97,14 @@
 
 body .jw-error,
 body .jwplayer.jw-state-error {
-  .jw-title .jw-title-primary {
-    white-space: normal;
+  .jw-title {
+    .jw-title-primary {
+      white-space: normal;
+    }
+    .jw-title-secondary {
+      display: block;
+      padding-top: 0.75em;
```

Comment:
displays description during error state. `padding-top: 0.75em;` resets padding for `.jw-title-secondary` that was `padding: 0.75em 0;`